Examples/README.md

```diff
@@ -3,4 +3,5 @@
 _These sample apps demonstrate Snap-ins features. They do not contain production-quality code and are not meant to be used in production apps._
 
 * [Snap-Ins SDK Hello World Example](./SnapinsSDKExample/): A simple Snap-ins SDK Hello World app that launches Knowledge, Cases, Chat, and SOS. This app uses a `KnowledgeViewAddition` to display floating action buttons.
-* [Snap-Ins SDK Chat Survey Example](./SnapinsChatSurvey/): A Live Agent Chat app that launches a chat session and then redirects the user to a survey page after the session.
\ No newline at end of file
+* [Snap-Ins SDK Chat Survey Example](./SnapinsChatSurvey/): A Live Agent Chat app that launches a chat session and then redirects the user to a survey page after the session.
+* [Snap-ins SDK Kotlin Example](./SnapinsSDKKotlinExample/): A full implementation sample application of the Service SDK (Knowledge, Case, Live Agent Chat and SOS) written in Kotlin.
```

Comment:
Use Title Case (for the I in Snap-Ins) for the product name before the colon... "Snap-Ins SDK Kotlin Example". And then instead of Service SDK in the body, say "Snap-ins SDK".